yourls/content.md

```diff
@@ -10,29 +10,42 @@ YOURLS is a set of PHP scripts that will allow you to run Your Own URL Shortener
 
 ## Start a `%%IMAGE%%` server instance
 
-```console
-$ docker run --name some-%%REPO%% --link some-mysql:mysql \
-    -e YOURLS_SITE="https://example.com" \
-    -e YOURLS_USER="example_username" \
-    -e YOURLS_PASS="example_password" \
-    -d %%IMAGE%%
+```bash
+docker run \
+    --name some-%%REPO%% \
+    --detach \
+    --link some-mysql:mysql \
```

Comment:
Just FYI, `--link` is deprecated and we removed it from most of our docs intentionally; see https://github.com/docker-library/docs/pull/1441

Reply:
(To be more clear: you're welcome to leave `--link` in here if you want, but I'd suggest swapping to `--network some-network` or similar.)